src/Console/Command/ShowCommand.php

```diff
@@ -40,16 +43,30 @@ public function __construct(Store $store, EventSerializer $serializer, Aggregate
     protected function configure(): void
     {
         $this
-            ->addArgument('aggregate', InputArgument::REQUIRED, 'aggregate name')
-            ->addArgument('id', InputArgument::REQUIRED, 'aggregate id');
+            ->addArgument('aggregate', InputArgument::OPTIONAL, 'aggregate name')
+            ->addArgument('id', InputArgument::OPTIONAL, 'aggregate id');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $console = new OutputStyle($input, $output);
 
-        $aggregate = InputHelper::string($input->getArgument('aggregate'));
-        $id = InputHelper::string($input->getArgument('id'));
+        $aggregate = InputHelper::nullableString($input->getArgument('aggregate'));
+        while ($aggregate === null) {
```

Comment:
Isnt an `if` sufficient for both arguments?

Reply:
If you hit just enter you will be asked again until you cancel the command, or enter something.

Reply:
I'm not sure if this approach or a error message which hints the user that he needs to choose one option to proceed is better DX.

But lets keep it with the requestioning, maybe with an additional text hinting the user that no input was given? This is up to you if you want to put more effort into it. 

A Testcase covering that the arguments are now optional would be great.

Reply:
Good point, will add a hint and test 😉